aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -45,6 +91,7 @@
       break;
     }
   }
+  _mtl_indexing_library = nil;
```

Comment:
> One needs to either initialize it to nil here or in MPSDevice constructor (similar to _mtl_device) otherwise it will lead to random crashes, similar to the ones I've fixed in https://github.com/pytorch/pytorch/pull/78136

Here `_mtl_indexing_library` is initialized (not released)

Reply:
Fair point, this needs to be released properly.